dashboard/client/src/pages/dashboard/node-info/features/GPU.tsx

```diff
@@ -133,7 +133,7 @@ const WorkerGPUEntry: React.FC<WorkerGPUEntryProps> = ({ resourceSlot }) => {
 };
 
 const WorkerGPU: WorkerFeatureRenderFn = ({ worker }) => {
-  const workerRes = worker.coreWorkerStats[0].usedResources;
+  const workerRes = worker.coreWorkerStats[0]?.usedResources;
```

Comment:
the `?` propagate the `undefined` value so if `coreWorkerStats` is empty, `workerRes` will be `undefined` as well.